safe/safe.go

```diff
@@ -0,0 +1,12 @@
+package safe
+
+import (
+	"github.com/bndw/pick/backends"
+)
+
+type Safe struct {
+	backend  backends.Backend   `json:"-"`
+	Accounts map[string]Account `json:"accounts"`
+}
+
+var _password []byte
```

Comment:
Do you have a recommendation on somewhere else to stick it? It's used again if `safe.Save` is called and I'd prefer not to prompt for it again. A private attribute on `Safe`?